### PR TITLE
feat(ui): Add banner to broadcast messages

### PR DESCRIPTION
**Pull request by rudra-superrr**
Signed-off-by: rchopra <rudra.chopra@siemens.com>

Issue:  Add banner to broadcast messages (#1595)

Description : This functionality can be edited by Admins only.

### How To Test?

1) Click on 9 dots on the top right section of the page
2) Go to control panel -> click custom fields under configuration
3) Select User under Resource section
4) There you will see BannerMessage custom field.
5) Click on three dots to edit this field. Here you can enter message that you want to be displayed.
6) Right click on three dots and give view permission to all sw360 users/roles and change other permissions too. (Refer the ss for this step)

### Screenshots

controlPanel
Click custom field under control panel section

User
Select User under resource section

bannerMessage
Banner Message custom field should look like this

Permissions
Give view permission to all sw360users/roles and make changes in other permissions too

welcomePage
Banner message on top of the page

### CODE REVIEW

**Comment by nikkuma7** [comment]
test successful.

image

File: frontend/liferay-theme/src/main/webapp/templates/portal_normal.ftl [hunk 1]
```diff
@@ -38,9 +38,15 @@
...
 <#assign login_css = is_signed_in?then("signed-in", "not-signed-in") />
+<#assign liferay_user = themeDisplay.getUser() />
+<#assign js_folder = theme_display.getPathThemeJavaScript() />
+<#assign js_banner_file = htmlUtil.escape(portalUtil.getStaticResourceURL(request, "${js_folder}/banner.js")) />
+
 <html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
 
 <head>
+	<script type="text/javascript" src ="${js_banner_file}" > </script>
+
```
Comment (akapti) [inline frontend/liferay-theme/src/main/webapp/templates/portal_normal.ftl:49]: Please remove this script tag, as banner JS has been removed. also at line `42` and `43`.

**Review by akapti — CHANGES_REQUESTED** [review]
please incorporate the review comment.

**Comment by akapti** [comment]
One suggesting is to use the bootstrap `alert` CSS instead of creating own CSS for `updateMessage` div.

The message will look something like this if we user bootstrap `alert-info` class:
image


in case we user `alert-warning` class:
image

**Comment by afsahsyeda** [comment]
Hi @akapti ,

I have incorporated the changes. Please check.
image

File: frontend/liferay-theme/src/main/webapp/templates/portal_normal.ftl [hunk 2]
```diff
@@ -38,6 +38,9 @@
 
 <#assign preferences = freeMarkerPortletPreferences.getPreferences({"portletSetupPortletDecoratorId": "sw360", "destination": "/search"}) />
 <#assign login_css = is_signed_in?then("signed-in", "not-signed-in") />
+<#assign liferay_user = themeDisplay.getUser() />
+<#assign js_folder = theme_display.getPathThemeJavaScript() />
```
Comment (akapti) [inline frontend/liferay-theme/src/main/webapp/templates/portal_normal.ftl:42]: please remove this variable as it's never used.

**Review by akapti — CHANGES_REQUESTED** [review]
please address review comment

**Comment by afsahsyeda** [comment]
Hi @akapti ,
Please check now.

**Review by akapti — APPROVED** [review]
Looks good.

**Pull request merged**